### ISSUE TITLE
lelkesedés

### ISSUE THREAD

**Issue by sassbalint** [badicsalex/torvenyek#2]
(Nem egy tipikus issú, mert nem feladat...)

Fantasztikusan jó az egész ötlet, cél, megvalósítás. :smiley:

Hasonló jellegű apró próbálkozásaim:
https://github.com/sassbalint/vagyonnyilatkozat
https://github.com/sassbalint/mta-itm-tv

Az esetleges jövőbeli együttműködést sem kizárva üdv.

---

_update:_ Izgalmas kérdés, hogy hogy történt a nyelvi feldolgozás.
Valamint: sztem difftoolnak megteszi akár maga a github.

**Comment by badicsalex** [comment]
A feldolgozást a https://github.com/badicsalex/hun_law csinálta, egészen konkrétan a definiált nyelvtan a munkagép:
https://github.com/badicsalex/hun_law/blob/master/hun_law/grammar/grammar.ebnf

A https://github.com/sassbalint/mta-itm-tv érdekes ötlet, igazából annak semmi akadálya nincs, hogy törvényjavaslatokra is elő lehessen állítani automatikusan az új állapotot, csak meg kell írni a PDF parsert a parlamenti iromány formátumra.

(Sajnos plusz probléma az 1994-es törvények kinyerése az akkori Magyar Közlönyből az most _kimondottan_ nem támogatott, csak a 2009 utáni formátum)

**Comment by badicsalex** [comment]
Ja, és difftoolnak a github csak a kisebb törvényekre jó, mert a nagy fájlokat nem akarja megjeleníteni.
